ui/packages/catalog-ui-search/src/main/webapp/component/search-form-editor/search-form-editor.less

```diff
@@ -0,0 +1,18 @@
+@{customElementNamespace}search-form-editor {
+  > .search-form-editor-title {
+    width: 34%;
+    vertical-align: top;
+    float: left;
+  }
+  > .search-form-editor-query-input {
+    width: 34%;
+    height: 98%;
+    clear: left;
+    overflow: scroll;
+    float: left;
+  }
+  > .search-form-editor-map {
+    margin-left: 34%;
+    height: 100vh;
```

Comment:
Won't 100vh take up the full height of the viewport?  Not sure that's what we would want.